packages/catalog-search/src/SearchBox.jsx

```diff
@@ -155,6 +164,9 @@ export const SearchBoxBase = ({
         value={defaultRefinement}
         onSubmit={handleSubmit}
         onClear={handleClear}
+        onFocus={(query) => {
```

Comment:
can I get a tldr on why we're setting the query on focus on top of on input change?

Reply:
the `onChange` handles input changes in the search field, where as the `onFocus` is used to handle a user's focus on the component I.e. when the user clicks or tabs into the search field.